boards/arm64/mimx8mn_evk/mimx8mn_evk_a53_smp.dts

```diff
@@ -31,7 +31,7 @@
 	};
 
 	sram0: memory@93c00000 {
-		reg = <0x93c00000 DT_SIZE_M(1)>;
+		reg = <0x93c00000 DT_SIZE_M(16)>;
```

Comment:
There is no harm in this, but we are now using the evk board in the boards-zephyr repo....